server/utils/addressToLines.ts

```diff
@@ -0,0 +1,22 @@
+import PrisonerContactSummary = contactsApiClientTypes.PrisonerContactSummary
+
+const addressToLines = ({
+  flat,
+  premise,
+  street,
+  area,
+  city,
+  county,
+  postalCode,
+  country,
+}: Partial<PrisonerContactSummary>): string => {
+  let lineOne = [premise, street].filter(s => s).join(', ')
+  if (flat) {
+    lineOne = `Flat ${flat}, ${lineOne}`
+  }
+  const addressArray = [lineOne, area, city, county, postalCode, country].filter(s => s).join('<br />')
```

Comment:
There is still HTML markup `<br/>` in the utils functions. I think markup should be limited to the presentation layer, though I know it isn't in every service. However, converting an address to an array of lines and passing this array of lines into the nunjuncks template gives an opportunity to present it there however each form wants to do it.